tests/Doctrine/ODM/MongoDB/Tests/Functional/MemoryUsageTest.php

```
@@ -25,6 +24,8 @@ class MemoryUsageTest extends BaseTest
 {
     /**
      * Output for jwage "Memory increased by 14.09 kb"
+     *
+     * @doesNotPerformAssertions
```

Since you've found this test: I wonder if it's needed :D I mean nobody looks at tests that did not fail explicitly

---

And this one won't be ending with an exception either

---

Those tests are excluded from the test run in `phpunit.xml.dist`:
```xml
    <groups>
        <exclude>
            <group>performance</group>
        </exclude>
    </groups>
```

I think their main purpose was to quickly test the performance and memory usage before adding phpbench (the test is dated 2014).

---

I think I we could keep these tests for now, for historical reasons and perhaps delete them in a separate PR if you don't mind. I'd like to keep the diff ass small as possible here.

---

Sure 👍